src/Controls/src/Core/Compatibility/Handlers/Shell/Android/ShellFlyoutRenderer.cs

```diff
@@ -183,9 +183,10 @@ protected override bool DrawChild(Canvas canvas, AView child, long drawingTime)
 				canvas.DrawRect(0, 0, Width, Height, _scrimPaint);
 			}
 
-			if (!FlyoutFirstDrawPassFinished && _flyoutContent != null)
+			if (!FlyoutFirstDrawPassFinished && _flyoutContent is not null)
 			{
-				if (child == _flyoutContent?.AndroidView)
+				// If the AndroidView property which is the DrawerLayout is initialized at this point, the Flyout first draw pass finished.
+				if (_flyoutContent?.AndroidView is not null)
```

Comment:
The original condition is only true after opened the Drawer at least once. For this reason, in the issue the comments share that it works if the menu is opened with a gesture or with the hamburger button at least once but not using directly the `FlyoutIsPresented` property.
The `FlyoutFirstDrawPassFinished` property prevents opening the Drawer before it is initialized with a property change to avoid incorrect states, but at this point, if the DrawerLayout has been initialized and the contents drawn we should be able to initialize it.